Source/Rows/TextAreaRow.swift

```diff
@@ -112,12 +114,19 @@ open class _TextAreaCell<T> : Cell<T>, UITextViewDelegate, AreaCell where T: Equ
             height = { cellHeight }
         }
 
+        if let percentage = textAreaRow.titlePercentage {
```

Comment:
I think we don't need the variable `titlePercentage` on the cell. Why don't use it from the row directly? That way if you decide to change it after the cell is set up, the constraints would be updated.